### PR TITLE
chore: flaky cypress tests

### PR DESCRIPTION
**Pull request by nsbarsukov**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [X] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Refactoring
- [ ] Code style update
- [X] Build or CI related changes
- [ ] Documentation content changes

## What is the current behavior?
code-highlight-cypress

## Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

### CODE REVIEW

File: projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts [hunk 1]
```diff
@@ -14,10 +17,16 @@ describe('Code blocks', () => {
     it('API page', () => {
         cy.tuiVisit('/components/line-clamp/Setup');
 
+        waitCodeBlockHighlight();
```
Comment (splincode) [inline projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts:20]: `cy.waitCodeBlockHighlight()`

File: projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts [hunk 2]
```diff
@@ -5,6 +5,9 @@ describe('Code blocks', () => {
         it(tabName, () => {
             cy.tuiVisit('/components/line-clamp');
             cy.get('#basic tui-tabs-with-more [tuiTab]').contains(tabName).click();
+
+            waitCodeBlockHighlight();
```
Comment (splincode) [inline projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts:9]: `cy.waitCodeBlockHighlight()`

File: projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts [hunk 3]
```diff
@@ -14,10 +17,16 @@ describe('Code blocks', () => {
...
+
         cy.get('tui-doc-code').each(($el, index) => {
             cy.wrap($el)
                 .wait(DEFAULT_TIMEOUT_BEFORE_ACTION)
                 .matchImageSnapshot(`02-${index}-API-page`);
         });
     });
 });
+
+function waitCodeBlockHighlight(): void {
```
Comment (splincode) [inline projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts:30]: please, you can move it to utils

Comment (nsbarsukov) [reply projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts]: ✅

File: projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts (hunk 2 above)
Comment (nsbarsukov) [reply projects/demo-integrations/cypress/tests/addon-doc/code.spec.ts]: ✅

**Review by splincode — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**